plugins/mips/mips_multiply.ml

```diff
@@ -0,0 +1,127 @@
+open Mips.Std
+
+(* MUL rd, rs, rt
+ * Multiply Words Signed, Low Word, MIPS32
+ * Page 351 *)
+let mul cpu ops =
+  let rd = signed cpu.reg ops.(0) in
+  let rs = signed cpu.reg ops.(1) in
+  let rt = signed cpu.reg ops.(2) in
+  RTL.[
+    rd := low word (rs * rt);
```

Comment:
double checking, @gitoleg are the `low`, `high`, and the rest of word projectors, updated wrt the MIPS ordering?